spec/jobs/in_person/enrollments_ready_for_status_check/enrollment_pipeline_spec.rb

```diff
@@ -0,0 +1,334 @@
+        expect(err_extra).to eq(extra)
+      end
+      expect(enrollment_pipeline.process_message(sqs_message)).to be(false)
+    end
+
+    context 'reports error and returns false' do
+      it 'SQS message is not JSON' do
+        allow(sqs_message).to receive(:body).and_return('not json')
+        expect_error(JSON::ParserError.new("unexpected token at 'not json'"), sqs_message_id:)
+      end
+
+      it 'SQS message body is not a hash' do
+        allow(sqs_message).to receive(:body).and_return('abcd'.to_json)
+        expect_error('SQS message body is not valid SNS payload', sqs_message_id:)
+      end
+
+      it 'SNS message is missing MessageId' do
+        allow(sqs_message).to receive(:body).and_return(
+          {
+            Message: 'abcd',
+          }.to_json,
+        )
+        expect_error('SQS message body is not valid SNS payload', sqs_message_id:)
+      end
+
+      it 'SNS message is missing Message' do
+        allow(sqs_message).to receive(:body).and_return(
+          {
+            MessageId: sns_message_id,
+          }.to_json,
+        )
+        expect_error('SQS message body is not valid SNS payload', sqs_message_id:)
+      end
+
+      it 'SNS "Message" field is not JSON' do
+        allow(sqs_message).to receive(:body).and_return(
+          {
+            MessageId: sns_message_id,
+            Message: 'not json',
+          }.to_json,
+        )
+        expect_error(
+          JSON::ParserError.new("unexpected token at 'not json'"), sqs_message_id:,
+                                                                   sns_message_id:
+        )
+      end
+
+      it 'SES message is not a hash' do
+        allow(sqs_message).to receive(:body).and_return(
+          {
+            MessageId: sns_message_id,
+            Message: 'abcd'.to_json,
+          }.to_json,
+        )
+        expect_error(
+          'SNS "Message" field is not a valid SES payload', sqs_message_id:,
+                                                            sns_message_id:
+        )
+      end
+
+      it 'SES message is missing "content" key' do
+        allow(sqs_message).to receive(:body).and_return(
+          {
+            MessageId: sns_message_id,
+            Message: {
+              mail: {},
+            }.to_json,
+          }.to_json,
+        )
+        expect_error(
+          'SNS "Message" field is not a valid SES payload', sqs_message_id:,
+                                                            sns_message_id:
+        )
+      end
+
+      it 'SES message is missing "mail" key' do
+        allow(sqs_message).to receive(:body).and_return(
+          {
+            MessageId: sns_message_id,
+            Message: {
+              content: 'abcd',
+            }.to_json,
+          }.to_json,
+        )
+        expect_error(
+          'SNS "Message" field is not a valid SES payload', sqs_message_id:,
+                                                            sns_message_id:
```

Comment:
i think the alignment is easier to follow if they are all on different lines
```suggestion
          'SNS "Message" field is not a valid SES payload',
          sqs_message_id:,
          sns_message_id:,
```